Update extensions.md (#104)
tachiyomiorg/website: src/help/faq/extensions.md

- Popular alternatives are **MangaDex**, **MangaSee** and **Mangakakalot**.

::: tip Out of the loop?
Read more about the situation [here](https://mangarock.com/).
:::

#### Migrate to Tachiyomi
There is a tool to migrate from **MangaRock** to **Tachiyomi** courtesy of a member of our [Discord server](https://discord.gg/tachiyomi) by the same name as the creator, but a computer is required to finish the migration successfully.

View the program at [waicool20/mr2tachiyomi](https://github.com/waicool20/mr2tachiyomi).

> #### Restore backup from Manga Rock source

Reviewer: ```suggestion
#### Restore backup from the MangaRock source
```

> #### Restore backup from  the MangaRock source

Reviewer: ```suggestion
#### Restore a backup from the MangaRock source
```

Author: Done

> It's currently not possible to restore backup from Manga Rock officially, as Manga Rock source is dead. There's an unofficial method to restore backup all manga from Manga Rock source.

Reviewer: ```suggestion
It's not currently possible to restore backup from MangaRock officially, as the MangaRock source is dead. There is howeven an unofficial method to restore/backup all manga from the MangaRock source.
```

Author: All changes made

> 1. Uninstall Manga Rock extension.

Reviewer: ```suggestion
1. Uninstall the MangaRock extension.
```

> 1. Install Dummy Rock apk. Link: [Dummy Rock](https://cdn.discordapp.com/attachments/403520500443119619/663671637052489728/tachiyomi-en.dummyrock-v1.2.1-debug.apk)

Reviewer: ```suggestion
1. Install the [DummyRock](https://cdn.discordapp.com/attachments/403520500443119619/663671637052489728/tachiyomi-en.dummyrock-v1.2.1-debug.apk) `.apk`. 
```

> 1. Go to **[<MaterialIcon icon-name="extension"/> Extensions](/help/guides/extensions)** menu and press **Trust** to trust Dummy Rock.

Reviewer: ```suggestion
1. Go to **[<MaterialIcon icon-name="extension"/> Extensions](/help/guides/extensions)** menu and press **Trust** to proceed.
```

> 1. Restore Backup

Reviewer: ```suggestion
1. Restore backup.
```
